src/activities/quizzes/attempts/QuizAttemptsEntity.js

```diff
@@ -166,8 +166,8 @@ export class QuizAttemptsEntity extends Entity {
 	_hasAttemptConditionChanged(attemptCondition) {
 		const original = this.getAttemptConditionSubEntity(attemptCondition.attempt);
 		if (!original || !original.properties) return false;
-		if (!original.properties.min && attemptCondition.min) return true;
-		if (!original.properties.max && attemptCondition.max) return true;
+		if (!original.properties.min && attemptCondition.min !== undefined) return true;

```

Comment:
What is this checking for? `0`, `NaN` or `null` I suppose? If `min` is `undefined` this condition will evaluate to `false` and if it's `null` or `NaN` it will be `true`? Do we explicitly handle the `undefined` case elsewhere?

Reply:
If original min is `undefined` and the new min has a numeric value, then it has changed. Perhaps it is safer to be explicit and say: 
`if (original.properties.min === undefined && attemptCondition.min !== undefined) return true;`

Reply:
Ah ok I was confused about what we were actually checking there. In this case then if `original.properties.min` is `0` and `attemptConditions.min` is also `0` won't it return a false positive?

Reply:
`null` and `NaN` shouldn't be passed into this